Gestion des Articles (Massages) en lien avec la Firebase

Amestyale opened the pull request.

Commit:
Connexion à la Firebase

Connexion mise en place afin de se connecter à la firebase, fonction connecttodb() accessible depuis les autres classes. Les informations de connexion sont rangées dans le fichier con3ctt0d8ta8bse.json

Commit:
Connexion à la Firebase

Connexion mise en place afin de se connecter à la firebase, fonction connecttodb() accessible depuis les autres classes. Les informations de connexion sont rangées dans le fichier con3ctt0d8ta8bse.json

Commit:
Ajout d'un fichier type pour l'objet Massage

Fichier type au format JSON des items contenus dans la firebase, accessible dans resources/json/

Commit:
Ajout d'un fichier type pour l'objet Massage

Fichier type au format JSON des items contenus dans la firebase, accessible dans resources/json/

Commit:
Résolution de problèmes lors de la connexion

Commit:
temp

maeldebon commented on app/Http/Controllers/ArticleController.php:
```diff
@@ -1,14 +1,16 @@
+
 class ArticleController extends Controller
 {
     //
```
Pas sûr que ce soit utile de laisser ça 🙂

maeldebon commented on app/Services/DatabaseService.php:
```diff
@@ -0,0 +1,112 @@
+        return $database;
+    }
+    
+}
```
Il manque un linebreak

maeldebon commented on app/Services/con3ctt0d8ta8bse.json:
```diff
@@ -0,0 +1,15 @@
+    "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+    "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/firebase-adminsdk-cmjw0%40coincoininsolite-1cf37.iam.gserviceaccount.com"
+  }
+  
```
Il faudrait rajouter un linebreak aussi 😄

maeldebon commented on resources/json/produit_type.json:
```diff
@@ -0,0 +1,32 @@
+    "bienfaits" : "Tout plein partout",
+    "allergies" : "Les allergies c'est pour les faibles"
+
+}
```
Idem

maeldebon commented on resources/views/product.blade.php:
```diff
@@ -1,7 +1,9 @@
+    <p>This is massage {{ $massage['nom'] }}</p>
+@endforeach
+
 @endsection
```
Tout pareil 😉

maeldebon reviewed (changes requested):
Très bon boulot !

Commit:
Suppression d'une fonction de test

Suppression d'une fonction de test + rename de la fonction d'insertion. L'action d'insérer est laissée en commentée pour l'instant, afin d'éviter des push involontaires vers la base

Commit:
Suppression de commentairs vides, ajout de linebreaker
